src/components/Header/HeaderContact.js

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
-import {StyleSheet, Text, View} from 'react-native';
+import {Platform, StyleSheet, Text, View} from 'react-native';
 import PropTypes from 'prop-types';
 import ArrowLeftIcon from '../../../assets/icons/arrow-left.svg';
 import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {COLORS, SIZES} from '../../utils/theme';
 import GlobalButton from '../Button/GlobalButton';
+import dimen from '../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement on line 2 has been updated to include `Platform` from 'react-native'. This is used later in the code to conditionally render components based on the operating system. The rest of the changes in this hunk are just formatting and do not affect the functionality of the code.
```diff
- import {StyleSheet, Text, View} from 'react-native';
+ import {Platform, StyleSheet, Text, View} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->